chromium_src/third_party/blink/renderer/core/css/css_font_selector.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_CSS_CSS_FONT_SELECTOR_H_
+#define BRAVE_CHROMIUM_SRC_THIRD_PARTY_BLINK_RENDERER_CORE_CSS_CSS_FONT_SELECTOR_H_
+
+#define BRAVE_CSS_FONT_SELECTOR_H \
```

Comment:
`#define UpdateGenericFontFamilySettings bool AllowFontFamily(const AtomicString& family) override; UpdateGenericFontFamilySettings` ?

Reply:
gone as of https://github.com/brave/brave-core/pull/12234/commits/236ac4bfa72c8cb6e79cafdc1d50c34166adbed4